NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -17,7 +17,7 @@
   <PropertyGroup>
     <AssemblyName>Microsoft.ApplicationInsights.AspNetCore</AssemblyName>
     <LangVersion>7.2</LangVersion>
-    <TargetFrameworks>netstandard2.0;net451;net46;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net451;net46;net461;netstandard1.6</TargetFrameworks>
```

Comment:
Also if you are adding net461 - add this as a condition to all places net46 is used. For instance, in here:

```
#if NET451 || NET46
        /// <summary>
        /// SDK Version Prefix.
        /// </summary>
        public const string VersionPrefix = "ilf:";
#else
        /// <summary>
        /// SDK Version Prefix.
        /// </summary>
        public const string VersionPrefix = "ilc:";
#endif
```

Reply:
just added! Thanks for the help!